rbx_binary/src/tests/snapshots/rbx_binary__tests__util__axes__decoded.snap

```diff
@@ -6,6 +6,7 @@ expression: decoded_viewed
   name: ""
   class: ArcHandles
   properties:
+    Adornee: "[unknown ID]"
```

Comment:
Ah! I'm going to land a change so that these will end up as zeroes instead of being reported unknown ID.

Reply:
Ok! If you merge the master branch in, these values should be replaced with `"null"` explicitly, instead of an unknown value.

Reply:
Lookin good!